beacon_node/http_api/src/lib.rs

```diff
@@ -3601,27 +3601,6 @@ pub fn serve<T: BeaconChainTypes>(
             })
         });
 
-    // POST lighthouse/database/historical_blocks
-    let post_lighthouse_database_historical_blocks = database_path
```

Comment:
Should this be deprecated? or updated to work with blobs? not sure where it is or was ever used

Reply:
We used to use it for debugging weak subjectivity sync but it's no longer used and Michael said to go ahead and deprecate it.